backend/rules/rules.js

```diff
@@ -0,0 +1,119 @@
+const troopsRule = {
+  buildCost: 10,
+  buildSecond: 60,
+  level1: {
+    attack: 1,
+    defence: 1,
+    sustenance: 1,
+    upgradeCost: 20,
```

Comment:
`attack`, `defence` is always equals to the level, `sustenance` is always 1, these are a lot of unnecessary code which can be solved somehow like this:
attack: level,
defense: level,
sustenance: 1,

and you don't have to write them into every levels, just like the buildCost.

also what if we want to have 5 levels in the future? I suggest to define a maximum level at each object and to calculate whatever you can based on the level of the troop, building, etc, 
it will be much easier to rewrite in the future, also now there is a lot of code duplication which can be solved with this method

```suggestion
    upgradeCost: level * 10, // or something like this
```